.releaserc

```diff
@@ -0,0 +1,28 @@
+{
+    "branches": [
+        { "name": "main" },
+        "+([0-9])?(.{+([0-9]),x}).x",
+        { "name": "beta", "channel": "beta", "prerelease": true },
+        { "name": "alpha", "channel": "alpha", "prerelease": true },
+        { "name": "next", "channel": "next" }
+      ],
+      "plugins": [
+        "@semantic-release/commit-analyzer",
+        "@semantic-release/release-notes-generator",
+        "@semantic-release/changelog",
+         ["@semantic-release/exec", {
```

Comment:
This should exec the script to add the latest release version to the package.json before npm publish